buzzmobile/sense/lidar/lidar.py

```diff
@@ -1,24 +1,55 @@
 import cv2
 import numpy as np
+import constants
 
 import time
 
-image_height_px = 800
-image_width_px = 800
-image_height_m = 8
-image_width_m = 8
-pixels_per_m = 100
+from cv_bridge import CvBridge, CvBridgeError
 
 
-# lidarPoints is a list of tuples. i.e. [(x0, y0), (x1, y1), ...]
-def genLidarImage(lidarPoints):
+image_width_px = constants.image_height
```

Comment:
Agreed that using inline is better. We can also do `import constants as c` or something like that for readability.